components/ContactCard/ContactCardForm.test.js

```diff
@@ -0,0 +1,10 @@
+import { render } from "@testing-library/react";
+import ContactCardForm from "./ContactCardForm";
+
+test("renders name, email, and message input fields", () => {
+  const { getByTestId } = render(<ContactCardForm />);
+  expect(getByTestId("name-input")).toBeInTheDocument();
+  expect(getByTestId("email-input")).toBeInTheDocument();
+  expect(getByTestId("message-input")).toBeInTheDocument();
+  expect(getByTestId("submit-button")).toBeInTheDocument();
```

Comment:
Can you use `getByLabelText` instead of `data-testid`s? Test ids are not great. See: https://testing-library.com/docs/queries/about#priority

Reply:
You will need labels for your inputs. Please go ahead and add those as well. Only `placeholders` is not good for accessibility.